src/main/scala/firrtl/transforms/PropagatePresetAnnotations.scala

```diff
@@ -8,7 +8,9 @@ import firrtl.annotations._
 import firrtl.ir.{AsyncResetType, _}
 import firrtl.options.Dependency
 
-import scala.collection.mutable
+import scala.collection.mutable.{ ArrayBuffer, HashMap, HashSet }
```

Comment:
Why did you change this import? It is standard in the `firrtl` code base to only import `scala.collection.mutable` and then prefix all types with `mutable.`.